content/our_ambassadors.html

```diff
@@ -493,6 +493,26 @@ <h4>Laramie Lindsey</h4>
               </div>
           </div>
 
+          <div class="feature">
+            <div class="feature-body ambassadors">
+              <img src="img/luke.jpg" class="ambassador" />
+                <h4>Luke Pembleton</h4>
+            </div>
+            <div class="feature-hover border-color">
+                <p class="text-muted">
+                Luke has a PhD in Molecular Genetics from <a href="https://www.latrobe.edu.au/" target="_blank">La Trobe University</a> and <a href="https://agriculture.vic.gov.au/about/our-research" target="_blank">AVR</a>, Australia. He is currently a Genomic Breeding Scientist and Global Strategic Science Manager at <a href="https://www.barenbrug.com" target="_blank">Barenbrug</a>.
+                <br>
+                <img src="https://flagicons.lipis.dev/flags/4x3/au.svg" class="ambassador" width=15 length=15 />
+                <br>
+                <a href="https://github.com/lpembleton" target="_blank"><i class="fa fa-github fa-lg"></i></a>
+                <a href="https://genomic.social/@lwpembleton" target="_blank"><i class="fa-brands fa-mastodon"></i></a>
+                <a href="https://www.linkedin.com/in/lpembleton/" target="_blank"><i class="fa fa-linkedin fa-lg"></i></a>
+                <a href="https://twitter.com/lwpembleton" target="_blank"><i class="fa fa-twitter fa-lg"></i></a>
+                <a href="https://bsky.app/profile/lwpembleton.bsky.social" target="_blank"><i class="fa-solid fa-square"></i></a>
```

Comment:
Apparently there's no Blue Sky icon in Font Awesome yet, so that's why the square for now, right?

Reply:
Yeah. I think it's better to drop it for now.

Reply:
Cool, I believe I have removed that line now 👍